examples/split-screen/game.json

```diff
@@ -0,0 +1,149 @@
+{
```

Comment:
Could we have a text on each view with the corresponding player's name

Reply:
Sorry I don't understand your request

Reply:
Done 
![image](https://user-images.githubusercontent.com/1670670/145723207-2e05adee-f2f3-45a3-b3eb-52611705f48c.png)

Reply:
Ah, that's nice indeed, thanks :)